documentation/library_guide/oneDPL_2022.0_changes.rst

```diff
@@ -0,0 +1,40 @@
+oneDPL 2022.0 Changes
+#####################
+
+This page lists breaking changes in oneDPL 2022.0 from oneDPL 2021.
+
+Support of C++11 and C++14 has been discontinued. 
+
+The internal parameter n of discard_block_engine was changed
+ from int to std::size_t type to align implementation with
+ С++ std proposal https://cplusplus.github.io/LWG/issue3561.
+ This change lets users to utilize the full range of values for P and R
+ template parameters of discard_block_engine.
+ 
+ .. Note::
+
+ Note: it may require rebuilding users’ code.
+
+
+The following algorithms have been removed from dpl namespace
```

Comment:
Can you add another layer of bullets and add colons? Change to: 

The following algorithms have been removed from the dpl namespace:
- include/oneapi/dpl/functional:
  - using ::std::binary_function;
  - using ::std::unary_function;

The following algorithms are deprecated in C++17 and removed for C++20:
- include/oneapi/dpl/functional:
  - using ::std::binary_negate;
  - using ::std::not1;
  - using ::std::not2;
  - using ::std::unary_negate;
- include/oneapi/dpl/type_traits:
  - using ::std::is_literal_type;
  - using ::std::is_literal_type_v;
  - using ::std::result_of;
  - using ::std::result_of_t;

Reply:
Applied the suggestion.